spec/forms/training_details_form_spec.rb

```diff
@@ -48,18 +48,32 @@
         end
       end
 
-      context "duplicate" do
+      context "duplicate active trainee" do
         let!(:existing_trainee) { create(:trainee, trainee_id: "Test123") }
-        let(:trainee) { build(:trainee, provider: existing_trainee.provider, trainee_id: "TEST123") }
+        let(:trainee) { build(:trainee, provider: existing_trainee.provider, trainee_id: existing_trainee.trainee_id) }
 
         it "returns a duplicate error message" do
+          expect(existing_trainee.inactive?).to be(false)
```

Comment:
I think you can extract this line into a separate example:
```suggestion
it "existing trainee remains active" do
  expect(existing_trainee.inactive?).to be(false)
end 
```